playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
> What is the plan for looking up the latest with containerized installs? Just pull the latest tag? If so, I'm good with that approach, we'd just need to keep in mind that we'd need to take care when testing this internally where the latest tags are broken (which I'm fine with, we'd just need to make sure it's documented somewhere).

Yeah just pulling latest tag, if we're ruling out hard coding in openshift-ansible I can't think of another reasonable option. Internal testers will have several options for overriding at least.

> I can live with that, though doing that in a way that wouldn't require pre-loading the host facts may prove difficult. The easiest way I can think of to do that would be to use vars on the role invocation, in which case the template would be evaluated at the time it is used by a task within the role.

I think facts are already loaded at the point we start applying these roles but I'll keep an eye out for trouble here in case I missed something.